addons/payment_payumoney/README.md

```diff
@@ -0,0 +1,26 @@
+# PayUmoney
```

Comment:
No `module history` ?

Reply:
There was probably some, but it's long gone :sweat_smile: